Please take care of the flake8 error: 

```
purchase_product_variants/models/procurement_order.py:25:33: E128 continuation line under-indented for visual indent
```

--- a/purchase_product_variants/__openerp__.py
+++ b/purchase_product_variants/__openerp__.py
@@ -33,6 +33,7 @@
         "Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>",
         "Ana Juaristi <ajuaristio@gmail.com>",
     ],
+    "license": "AGPL-3",
     "category": "Custom Module",
     "website": "http://www.odoomrp.com",
     "summary": "",

--- a/purchase_product_variants/models/procurement_order.py
+++ b/purchase_product_variants/models/procurement_order.py
@@ -21,8 +21,9 @@ class ProcurementOrder(models.Model):
             product = self.env['product.product'].with_context(ctx).browse(
                 vals['product_id'])
             vals['product_template'] = product.product_tmpl_id.id
-            vals['product_attributes'] = [(0, 0, x) for x in
-                                product._get_product_attributes_values_dict()]
+            vals['product_attributes'] = \
+                [(0, 0, x) for x in
+                 product._get_product_attributes_values_dict()]
             vals['name'] = self.env[
                 'purchase.order.line']._get_product_description(
                 product.product_tmpl_id, product, product.attribute_value_ids)

